Handle WebAuth on Extreme Controller without web form (#7091)
inverse-inc/packetfence: docs/network/networkdevice/extreme_networks_wireless.asciidoc

// to display images directly on GitHub
ifdef::env-github[]
:encoding: UTF-8
:lang: en
:doctype: book
:toc: left
:imagesdir: ../../images
endif::[]

////

    This file is part of the PacketFence project.

    See PacketFence_Network_Devices_Configuration_Guide.asciidoc
    for authors, copyright and license information.

////

//=== Extreme Networks

==== Access points AP305C (managed by Extreme Cloud IQ Controller)

> In such deployment, PacketFence communicate directly with access

Reviewer: ```suggestion
In such deployment, PacketFence communicates directly with the access
```

> points using RADIUS. Extreme Cloud IQ controller is only used to configure

Reviewer: ```suggestion
points using RADIUS. The Extreme Cloud IQ controller is only used to configure
```

access points in a central place.

===== Web authentication

====== Extreme Cloud IQ Controller

> On Extreme Cloud IQ Controller, you should have builtin IP Firewall policies:

Reviewer: ```suggestion
On the Extreme Cloud IQ Controller, there should already be two built-in IP Firewall policies:
```

* Redirect-only policy: this policy must have "redirect" rules
* Internet-access-only policy

> On Extreme Cloud IQ Controller, create two user profiles:

Reviewer: ```suggestion
On the Extreme Cloud IQ Controller, create two user profiles:
```

* Registration
** VLAN: 5
** Firewall rules: Enabled
** IP Firewall Name: Redirect-only policy
** Redirect URL: http://192.168.1.5/Extreme::AP
* Guest
** VLAN: 5
** Firewall rules: Enabled
** IP Firewall Name: Internet-access-only policy

> On Extreme Cloud IQ Controller, create a wireless network with following configuration:

Reviewer: ```suggestion
Still, on the Extreme Cloud IQ Controller, create a wireless network with the following configuration:
```
